scripts/jarvis.js

```diff
@@ -1,4 +1,5 @@
 const fs = require('fs');
+const ora = require('ora');
```

Comment:
Could we move the `ora` require to the scope of the function that executes it? The reason we don't have any imports on the root of the file, is that the file will still be executable even if `node_modules` was not installed yet. Mainly so we can have a single entry point for new contributors (jarvis also does pnpm install if node_modules are not installed).

It's a bit confusing nonetheless, so maybe we could even move the 2 piece of code to different files so we have a node-native only file and the other file that could import things from npm. wdyt?

Reply:
You are right. I totally missed that pattern when doing the fix. Will change.